packages/xchain-ethereum/src/client.ts

```diff
@@ -118,36 +124,22 @@ export default class Client implements XChainClient, EthereumClient {
     ethplorerUrl = 'https://api.ethplorer.io',
     ethplorerApiKey = 'freekey',
     explorerUrl,
-    phrase,
+    phrase = '',
+    rootDerivationPaths = {
+      mainnet: `m/44'/60'/0'/0/`,
+      testnet: `m/44'/60'/0'/0/`, // this is INCORRECT but makes the unit tests pass
```

Comment:
Why is this incorrect? mainnet and testnet derivation paths are the same for ETH mainnet or testnet. 

https://iancoleman.io/bip39/